lib/camp.js

```diff
@@ -351,6 +351,7 @@ function SecureCamp(opts) {
 inherits(SecureCamp, https.Server);
 
 Camp.prototype.handle = SecureCamp.prototype.handle =
+Camp.prototype.use = SecureCamp.prototype.use =
```

Comment:
Middlewares are just functions handling `(augmentedRequest, augmentedResponse, next)`, so they should pretty much work the same for express and camp. Subtle differences are things like where parameters are located (e.g. for a path like `/users/:id`, express will have `req.params.id` vs camp's `req.data.id`), but then again some middlewares depend on other middlewares (e.g. using `req.cookies` in express assumes that you did something like `app.use(cookieParser())` before) so for more involved middlewares you always need to know what you're doing / what your `req` object has. TL;DR aliasing `handle` as `use` makes it clearer that it accepts middleware, and I guess it will simple work for most server middleware out there.